open-sauced/app
# fix: fixed dropdown styling

I decreased the size of the modal as well as the dropdown in desktop. However, I increased the size on mobile to make it more inline with the mobile styling.

fix #4063

## Description

This PR fixes bug #4063 by decreasing the size of the dropdown selector on desktop. I also took it upon myself to make the mobile styles more inline by _increasing_ the size of the mobile dropdown. You can see these changes in the video below.

## Related Tickets & Documents
Fixes #4063 

## Mobile & Desktop Screenshots/Recordings
[Watch the video](https://www.youtube.com/watch?v=paWFZ5t_p3E) I made for this fix.
### Desktop
**Before**
![Arc Repositories  LetsGetTechnicalgridiron-survivor-000195](https://github.com/user-attachments/assets/4ec108cf-5ef3-474e-aeda-379a61cbe140)

**After**
![Arc Repositories  LetsGetTechnicalgridiron-survivor-000197](https://github.com/user-attachments/assets/ff610f40-ba81-492f-b56a-580be3a2c92f)

### Mobile
**Before**
![Arc Repositories  LetsGetTechnicalgridiron-survivor-000199](https://github.com/user-attachments/assets/98c6c1b6-93f6-4d6e-a249-5d46d54ee8e0)

**After**
![Arc Repositories  LetsGetTechnicalgridiron-survivor-000201](https://github.com/user-attachments/assets/5fe610c3-a117-455b-86f6-cfe3b94590e3)

## Steps to QA
Visit any repo of your liking and click the "Add to workspace" button. Please check it in mobile as well.


## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?
![an-older-man-wearing-glasses-and-a-maroon-sweater-is-smiling](https://github.com/user-attachments/assets/eada9009-9c2c-493d-a98c-51eeb17ef73a)
